workshop organization/script presentation.md

```diff
@@ -0,0 +1,20 @@
+# Script for the introduction talk
+
+* Welcoming everybody’s
+* I am <your name>, I am a ... and one of the organizer of codebar Barcelona
+* present the other codebar barcelona chapter organizers present or not present
+* present codebar, what we want and how we plan to achieve it
+* inscription at codebar.io, slack channel, feedback
+* Thanks the <host>, host talk is they want
+* How does this workshop works.
+* Coaches (thanks them)
+* Student (thanks them too)
+* ![Student guide](http://tutorials.codebar.io/general/setup/tutorial.html) or ![Coaching guide](https://codebar.io/effective-teacher-guide)
+* Presentation rounds:
+  * Name
+  * where are you from
+  * what are you doing
+  * what do you want to achieve
+  * answer the silly question
```

Comment:
I found this in a Trello card while cleaning up. Worth adding it for inspiration: https://museumhack.com/list-icebreakers-questions/